web/src/main/java/com/epam/rft/atsy/web/controllers/ApplicationStateController.java

```diff
@@ -137,8 +137,13 @@ public ModelAndView loadPage(@RequestParam Long applicationId,
 
   private boolean isRepresentationsListFirstStateEquals(
```

Comment:
You have changed the method implementation, but the name of the method was not renamed.